OpenRA.Mods.Common/Graphics/RangeCircleRenderable.cs

```diff
@@ -43,13 +47,25 @@ public RangeCircleRenderable(WPos centerPosition, WDist radius, int zOffset, Col
 		public IFinalizedRenderable PrepareRender(WorldRenderer wr) { return this; }
 		public void Render(WorldRenderer wr)
 		{
-			var wlr = Game.Renderer.WorldLineRenderer;
-			var oldWidth = wlr.LineWidth;
-			wlr.LineWidth = 3;
-			wr.DrawRangeCircle(centerPosition, radius, contrastColor);
-			wlr.LineWidth = 1;
-			wr.DrawRangeCircle(centerPosition, radius, color);
-			wlr.LineWidth = oldWidth;
+			DrawRangeCircle(wr, centerPosition, radius, 1, color, 3, contrastColor);
+		}
+
+		public static void DrawRangeCircle(WorldRenderer wr, WPos centerPosition, WDist radius,
+			float width, Color color, float contrastWidth, Color contrastColor)
```

Comment:
It is the width of the range circle that is being drawn by `DrawRangeCircle`.  I don't see how that can be more verbose without being redundant.